With IPD4 illumos changed man page sections to a more common standard.

--- sudo-1.9.11p1/configure.ac.orig	2022-06-08 16:32:57.000000000 +0000
+++ sudo-1.9.11p1/configure.ac	2022-06-08 22:38:26.840283377 +0000
@@ -1805,9 +1805,9 @@
 		if test "$with_AFS" = "yes"; then
 		    AFS_LIBS="-lc -lucb"
 		fi
-		: ${mansectsu='1m'}
-		: ${mansectform='4'}
-		: ${mansectmisc='5'}
+		: ${mansectsu='8'}
+		: ${mansectform='5'}
+		: ${mansectmisc='7'}
 		test -z "$with_pam" && AUTH_EXCL_DEF="PAM"
 		AC_CHECK_FUNCS([priv_set], [PSMAN=1])
 		;;
